go.mod

```diff
@@ -84,3 +86,5 @@ replace github.com/gocql/gocql => github.com/grafana/gocql v0.0.0-20200605141915
 
 // Using a 3rd-party branch for custom dialer - see https://github.com/bradfitz/gomemcache/pull/86
 replace github.com/bradfitz/gomemcache => github.com/themihai/gomemcache v0.0.0-20180902122335-24332e2d58ab
+
+replace github.com/thanos-io/thanos => github.com/Oghenebrume50/thanos v0.13.1-0.20210704215112-03ccac2cf15a
```

Comment:
For `replace` lines, always add a comment saying why we need it, and what needs to change before it can be removed.

If this is just for test purposes, please mark the PR as 'draft'.